data/css/modules/card/_media.scss

```diff
@@ -1,35 +1,30 @@
 .CardMedia {
-    padding: 10px;
     background-color: white;
 
-    separator {
-        background-color: $black-20-color;
-        min-height: 1px;
-    }
-
     &__caption {
-        font-size: .81em;  /* 13pt * 1/12 * 0.75 DPI correction factor */
-        font-family: $display-font;
+        font-size: 20px;
+        font-family: Merriweather;
```

Comment:
Just a reminder of what we talked on slack about using specific fonts in modules partials, we should ideally replace it with the proper scss font variables unless we really want all of our apps to use this specific font.